README

```diff
@@ -231,6 +231,12 @@ options.
        available for private use (i.e., with contexts that enable the
        SHMEM_CTX_PRIVATE option).
 
+    SHMEM_OFI_STX_DISABLE_PRIVATE (default: 0)
+       Disallow private contexts from having exclusive STX access.  Setting this
+       to 1 may improve performance for applications using many private
+       contexts, especially if the number of threads that use private contexts
+       is greater than SHMEM_OFI_STX_MAX.
```

Comment:
Default should be "off".  For boolean env vars, setting them to _any_ value (including the empty string) enables them.  In terms of verbiage, here's a suggestion:

Disable STX privatization.  Enabling this may improve load balance across transmit resources, especially in scenarios where the number of contexts exceeds the number of STXs.

Reply:
👍